examples/Components/Routes/VueSlots/StaffList/DefaultBlockSlotExtension.js

```diff
@@ -0,0 +1,26 @@
+import Node from '../../../../../packages/tiptap/src/Utils/Node'
+
+export default class extends Node {
+  get name() {
+    return 'stafflistdefaultblockslot'
+  }
+
+  get schema() {
+    return {
+      content: 'block*',
+      group: 'block',
+      attrs: {
+        'v-slot:default': { default: 'slotProps' },
+      },
+      selectable: false,
+      draggable: false,
+      parseDOM: [{
+        tag: 'template[v-slot\\:default]',
+        getAttrs: dom => ({
+          'v-slot:default': dom.getAttribute('v-slot:default'),
+        }),
+      }],
+      toDOM: node => ['template', node.attrs, 0],
```

Comment:
I am not familiar with Vue, but wouldn't this clash with the DOM [template](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template).

Reply:
I'm not familiar enough with the inner workings of vue to give you an answer on that myself, but Vue does make use of `<template>` elements within its document structure.

I have used this element type successfully in Vue apps using tiptap by injecting the resulting output of `getHTML()` (that includes a `<template>` element) into a Vue component as so:

```
<component :is="injectedDocString"></component>
```